data/remnant missions.txt

```diff
@@ -495,3 +560,772 @@ mission "Remnant: Technology Available"
+		conversation
+			`The return trip to Viminal is much more interesting. The crew spend more time wandering your ship than they do in their bunks. Some spend most of their time down in engineering, while others hang out around the main guns or the cockpit. Only their remarkable aptitude at figuring out where to be in order to not interfere with normal operations keeps them from being a hindrance. As you open the hatch they disperse across the tarmac to the ships they have been assigned to, each one of them signing their thanks as they leave.`
+			`	A few minutes after they leave your comlink beeps, notifying you that a message from the prefect has arrived:`
+			`	"Captain <last>! Thank you for your help in defending Viminal and transporting those wounded. I have transferred a small token of our appreciation to you. That being said, perhaps you would appreciate this a bit more: the other prefects and I have agreed to grant you access to a wider range of ships. Our Albatross class warships, and Pelican class freighters will now be available to you. At the rate these raids are going, it is in everyone's best interests that you have access to a better warship."`
+			`	With a sense of finality the prefect makes a gesture that you've come to recognize as "respectful end of conversation."`
+
+event "Remnant Albatross"	
+	shipyard "Remnant"
+		"Albatross"
+		"Pelican"
+
+
+
+mission "Remnant: Salvage 4"
+	source "Viminal"
+	name "Track down the <npc>"
+	description "Track down and disable a Korath ship that escaped the raid on Viminal."
+	to offer
+		has "Remnant: Salvage 3: done"
+	on offer
+		conversation
+			`Walking into the cafeteria you immediately sense the buoyant mood. According to a report on the video screen very few Remnant ships were actually lost. Furthermore, of the ships that were lost, the escape pods worked perfectly, ensuring none of the crew were killed. Groups of crew are gathered around tables intermingled with the ground teams that keep their ships operational, and based on the snippets you can catch, they seem to be trading stories of crack shots and clever improvisations.`
+			`	Tali sits over at a corner booth with the prefect who met you at the dock and a couple other mechanics, and she waves you over. "Captain <last>! So, how do you like fighting with our fleet?" asks one of the mechanics.`
+			choice
+				`	"It seems odd to have a planetary defense fleet plundering."`
+				`	"It was a tough battle."`
+					goto next
+	
+			`	"I suppose it would," chants one of them. "We need access to their tech, though. Otherwise we'll fall behind." Judging by the finality of his statement, this is held to be common knowledge among the Remnant.`
+				goto next
+	
+			label next
+			`	While you're conversing with the group, the prefect suddenly glances down at his comlink, reads a message, then looks up at you. "It appears that a Korath ship survived the raid and has destroyed a few ships without taking cargo. Could you deal with it?`
```

Comment:
We should also be more consistent with our use of "com(m)s" and "com(m-)links". It appears that Michael is not terribly consistent himself in that manner.

Reply:
To note, comlink is only used once in game during the FW intro, while commlink is never used.

Reply:
And in the Remnant missions comlink is used five times, and now there are no instances of comm-link.